salt/cloud/clouds/profitbricks.py

```diff
@@ -6,7 +6,7 @@
 The ProfitBricks SaltStack cloud module allows a ProfitBricks server to
 be automatically deployed and bootstraped with Salt.
 
-:depends: profitbrick >= 3.0.0
+:depends: profitbrick >= 3.1.0
```

Comment:
Please document this change in the Oxygen release notes: https://github.com/saltstack/salt/blob/develop/doc/topics/releases/oxygen.rst